projects/plugins/jetpack/_inc/client/components/button/style.scss

```diff
@@ -8,201 +8,192 @@
 // ==========================================================================
 
 .dops-button {
```

Comment:
The current approach in Jetpack is to use the `rna` prop on the `Button` component to apply the custom `.is-rna` styles over the default dops button.

I think we are okay to commit these styles directly to the button component, so that we globally use the new button styles. But we'll just need to remember to clean up/unify the different approaches :+1:

Reply:
Ah, thanks! I'll check it out and clean up the code. Using .is-rna should be easier than what I did there